### DISCUSSION TITLE
How to query best selling in console

### DISCUSSION THREAD

**Discussion by jaymar-webriq** [swellstores/community#348] [Help] [answered]
Hello, I need help on how to query on console to get the best selling products and sort it by sales.
I use swell node backend and this query results will display on frontend, thanks

**Comment by sags95** [comment, marked as answer]
There is a `popularity` field on the product model that gets incremented after orders. So you can fetch products and sort by `popularity`

```
{
   "sort": "popularity desc"
}
```

If you want to pull in actual sales figures then you would then have to query the `/orders` endpoint as well
